cppcache/src/TcrConnection.cpp

```diff
@@ -917,12 +917,6 @@ char* TcrConnection::readMessage(size_t* recvLen,
 void TcrConnection::readMessageChunked(
     TcrMessageReply& reply, std::chrono::microseconds receiveTimeoutSec,
     bool doHeaderTimeoutRetries) {
-  int32_t messageType;
-  int32_t transactionId;
-  int32_t numberOfParts;
-  int32_t chunkLength;
-  int8_t flags = 0x0;
-
   auto headerTimeout =
       calculateHeaderTimeout(receiveTimeoutSec, doHeaderTimeoutRetries);
```

Comment:
Another long standing thing has been to fix variable names. The value of `receiveTimeoutSec` is no longer in seconds. ;)